lib/syskit/roby_app/in_process_tasks_manager.rb

```diff
@@ -0,0 +1,163 @@
+# frozen_string_literal: true
+
+module Syskit
+    module RobyApp
+        # A class API-compatible with {Syskit::RobyApp::RemoteProcesses::Client} but that
+        # creates components in-process
+        class InProcessTasksManager
+            # Exception raised if one attempts to do name mappings in an
+            # unmanaged process server
+            class NameMappingsForbidden < ArgumentError; end
+
+            # Fake status class
+            class Status
```

Comment:
I dont see the point of this class, on wait_termination you mention returning a mapping with the status but in fact you are returning a mapping with the process. Also, isnt it possible to get the actual status with the in process task if its terminated?

Reply:
That's ... a good catch.

Essentially, wait_termination does not return any status. This actually causes every deployment using this process manager to emit failed on stop. The unmanaged task manager has the same issue.

Reply:
Fixed